rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/360_date_histogram.yml

```diff
@@ -61,3 +112,103 @@ setup:
   - match: { aggregations.histo.buckets.8.doc_count: 1 }
   - match: { aggregations.histo.buckets.12.key_as_string: "2016-06-01T00:00:00.000Z" }
   - match: { aggregations.histo.buckets.12.doc_count: 1 }
+
+---
+"date_histogram and date_histogram_composite timezone":
+  - skip:
+      version: " - 8.1.99"
+      reason: bug fixed in 8.2.0
```

Comment:
Is this really fixed in 8.2?  I can understand not wanting to bisect all the way back to 7.2 to see where it got fixed, but if it works in 7.17, let's at least enable this for the LTS release, please.  That way it'll run with BWC stuff.  A test-only backport is probably a good idea too.